src/@types/parseable/api/savedFilters.ts

```diff
@@ -7,7 +7,7 @@ export type SavedFilterType = {
 	filter_id: string;
 	query: {
 		filter_type: 'sql' | 'builder';
-		filter_query?: string;
+		filter_query: string;
```

Comment:
existing saved filters might not have the filter_query . This type should it have as optional